UPGRADING.md

```diff
@@ -0,0 +1,24 @@
+- [Upgrading OpenSearch Python Client](#upgrading-opensearch-python-client)
+  - [Upgrading to >= 2.2.0](#upgrading-to->=-2.2.0)
```

Comment:
This link needs to be fixed here.

Reply:
Vamsi, I think that link is working. I checked in VS code it is pointing rightly. But when we see it in github page it is appearing like it is not working.